LICENSE

```diff
@@ -8,12 +8,83 @@ License text copyright (c) 2017 MariaDB Corporation Ab, All Rights Reserved.
 
 Parameters
 
-Licensor:             Babylon Labs, Ltd.
+Licensor:             Babylon Labs Ltd.
+
+Licensed Work:        Babylon Node
```

Comment:
Does this need to be same as repo name?

Reply:
with capitalized words?

Reply:
Adam didn't like this being just "Babylon" as it might be confusing, so I added Babylon Node.